db/migrate/20240109035854_create_observation_accuracy_validator.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateObservationAccuracyValidator < ActiveRecord::Migration[6.1]
+  def change
+    create_table :observation_accuracy_validators do | t |
```

Comment:
I think it would be good to have a column for `observation_accuracy_experiment_id` and to have a unique index on `user_id` and `observation_accuracy_experiment_id`. That way its clear which experiment these users are associated with, ensure that users are associated with a single experiment once, but that they can also be associated with different experiments (and be emailed about the individually) over time